src/main/java/salesforce/ui/pages/legalentity/LegalEntityPageAbstract.java

```diff
@@ -0,0 +1,18 @@
+package salesforce.ui.pages.legalentity;
```

Comment:
You should add copyright to match our teams conventions.

Reply:
okay